README-template.md

```diff
@@ -0,0 +1,90 @@
+# Welcome to the [TODO:Projectname] Project!
+
+<!-- Mission Statement -->
+<!-- More information about crafting your mission statement with examples -->
+<!-- https://contribute.cncf.io/maintainers/governance/charter/ -->
+
+[TODO: PROJECTNAME] is a [TODO: Type of Tool] that [TODO: Functions it
+performs]. [TODO: Reasons why these are needed and valuable]. [TODO:
+Implementation, strategy and architecture].
+
+[TODO: Additional paragraph describing your project (optional)]
+
+[TODO: PROJECTNAME] is hosted by the [MDN Web Docs](https://github.com/mdn).
+
+## Getting Started
+
+<!-- Include enough details to get started using, or at least building, the
+project here and link to other docs with more detail as needed.  Depending on
+the nature of the project and its current development status, this might
+include:
+* quick installation/build instructions
+* a few simple examples of use
+* basic prerequisites
+-->
+
+## Contributing
+
+Our project welcomes contributions from any member of our community. To get
+started contributing, please see our [Contributor Guide](CONTRIBUTING.md).
+
+## Scope
+
+<!-- If this section is too long, you might consider moving it to a SCOPE.md -->
+<!-- More information about creating your scope with links to examples -->
+<!-- https://contribute.cncf.io/maintainers/governance/charter/ -->
+
+### In Scope
+
+[TODO: PROJECTNAME] is intended to [TODO: Core functionality]. As such, the
+project will implement or has implemented:
+
+- [TODO: High-level Item 1]
+- [TODO: High-level Item 2]
+- [TODO: High-level Item 3]
+
+### Out of Scope
+
+[TODO: PROJECTNAME] will be used in a cloud native environment with other
+tools. The following specific functionality will therefore not be incorporated:
+
+- [TODO: Excluded function 1]
+- [TODO: Excluded function 2]
+
+[TODO: PROJECTNAME] implements [TODO: List of major features, existing or
+planned], through [TODO: Implementation
+requirements/language/architecture/etc.]. It will not cover [TODO: short list
+of excluded items]
+
+## Communications
+
+<!-- Fill in the communications channels you actually use.  These should all be public channels anyone
+can join, and there should be several ways that users and contributors can reach project maintainers.
+If you have recurring/regular meetings, list those or a link to a publicy-readable calendar so that
+prospective contributors know when and where to engage with you. -->
+
+[TODO: Details (with links) to meetings, mailing lists, Slack, and any other communication channels]
+
+- User Mailing List:
+- Developer Mailing List:
+- Slack Channel:
+- Public Meeting Schedule and Links:
+- Social Media:
+- Other Channel(s), If Any:
```

Comment:
This is useful, ways to report things like contributor / COC violations or even security things could be added. https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository#about-security-policies